.github/workflows/schedule_reporter.yml

```diff
@@ -0,0 +1,29 @@
+# Copyright 2025 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Schedule Reporter
+
+on:
+  schedule:
+    - cron: '0 6 * * *'  # Runs at 6 AM every morning
+
+jobs:
+  run_reporter:
+    permissions:
+      issues: 'write'
+      checks: 'read'
+      contents: 'read'
+    uses: ./.github/workflows/cloud_build_failure_reporter.yml
+    with:
+      trigger_names: "integration-test-pr-py39,integration-test-pr-py313"
```

Comment:
we do not want these to report on pull request builds, we should report on stable builds, continuous (merge to main) and periodic (nightly) builds....